pinot-core/src/test/java/org/apache/pinot/core/transport/QueryRoutingTest.java

```diff
@@ -62,7 +63,8 @@ public class QueryRoutingTest {
 
   @BeforeClass
   public void setUp() {
-    _queryRouter = new QueryRouter("testBroker", mock(BrokerMetrics.class));
+    // TODO(Vivek): Check if any testing can be done here.
```

Comment:
(nit) Suggest not leaving name based TODOs in the code

Reply:
Removed.